testcases/cve/cve-2012-0957.c

```diff
@@ -29,12 +29,15 @@
 #include "tst_test.h"
 #include "lapi/personality.h"
 
-static int check_field(char *bytes, size_t length, char *field)
+static struct utsname saved_buf;
+
+static int check_field(char *bytes, char *saved_bytes, size_t length,
+		       char *field)
```

Comment:
Have you run checkpatch.pl on this?

Reply:
@richiejp Yes. I checked with it and no error and warning are reported.

Reply:
@richiejp This indent comprises 2 tabs + 7 spaces.